src/App.jsx

```diff
@@ -4,13 +4,25 @@ import TodoList from './TodoList';
 import TodoForm from './TodoForm';
 
 function App() {
-  const [newTodo, setNewToDo] = useState('initialValue');
+  const [todoList, setTodoList] = useState([]);
+  const addToDoHandler = (title) => {
```

Comment:
Keep in mind that naming this function as it currently is is bad practice. Read my notes in previous PR. I suggest changing it.